common/git-hooks/pre-commit

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
```

Comment:
Does this work on Windows even if you don't have sh on your path (from git or some other source)?

Reply:
Just spoke with Pete offline and he says yes. He believes that git should sort that out before it invokes the hook

Reply:
This blog seems like it wouldn't work: https://www.tygertec.com/git-hooks-practical-uses-windows/ - but that's back in 2016. Maybe they fixed this?

Reply:
What do you think @pgonzal?

Reply:
As far as I know, Git Hooks work fine on Windows.